Sources/Swift/Integrations/SessionReplay/SentryBreadcrumbReplayConverter.swift

```diff
@@ -0,0 +1,83 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryBreadcrumbReplayConverter: NSObject {
```

Comment:
`h`: This class heavily relies on the implementation detail of classes such as `SentryBreadcrumbTracker`, `SentrySystemEventBreadcrumbs`, etc. I'm worried if we change something in our breadcrumbs implementation that we will break the logic here. Do you have suggestions to mitigate this risk?

Reply:
That's a good point, that I dont have an answer for other than we should be aware of it when review PRs.

Session replay is expecting a specific data struct different from the one used by our breadcrumbs, that's why we need the converter, the best approach would be strongly typed breadcrumbs, which we dont have.